pkg/domain/infra/abi/generate.go

```diff
@@ -60,9 +60,10 @@ func (ic *ContainerEngine) GenerateKube(ctx context.Context, nameOrIDs []string,
 				return nil, err
 			}
 		} else {
-			if len(ctr.Dependencies()) > 0 {
-				return nil, errors.Wrapf(define.ErrNotImplemented, "containers with dependencies")
-			}
+			// TODO: Test if this actually works, now that infra holds NS data, we need to support dependencies.
```

Comment:
I am uncomfortable with that.  Does it work? I think we should clarify before merging.

Reply:
Yep, this does in fact work. Removing the line causes all sorts of failures and the container, when started is in a proper state and so are all of its dependencies.

Reply:
Just left the todo there as a reminder in case something broke from a future fix, I will switch from a TODO to just a general comment